app/steps/medicaid/step_navigation.rb

```diff
@@ -2,6 +2,7 @@ module Medicaid
   class StepNavigation
     ALL = {
       "Introduction" => [
+        Medicaid::WelcomeController,
         Medicaid::IntroLocationController,
```

Comment:
do we need to add something to the view template for this controller that allows the person to go "Back"? I can't remember how we do the "clear session" on back for the first step.

Reply:
It automatically clears the session upon leaving. The static pages controller clears out the session info. And this first page has the "back" link enabled. Or are you saying this page should *not* have it enabled?